WordPress/src/main/java/org/wordpress/android/viewmodel/activitylog/ActivityLogViewModel.kt

```diff
@@ -151,6 +159,15 @@ class ActivityLogViewModel @Inject constructor(
         }
     }
 
+    fun dateRangePickerClicked() {
+        _showDateRangePicker.value = ShowDateRangePicker(initialSelection = currentDateRangeFilter)
```

Comment:
Tbh I prefer to keep the naming parameters since you don't need to navigate to the class and you can clearly see what you are setting. If the variable was called `initialSelection` I wouldn't have used it, but since it's called `currentDateRangeFilter` I still find the naming parameter useful. Wdyt?

Reply:
What you described makes sense! Please feel free to keep it as is. Sometime the ide would actually do that for you as well, giving you the named parameter for free, but this way it is more explicit. Usually, I am using named parameters as a builder, or when I want to change the order, but this makes sense as well.